src/glang/lib/std/json.c3

```diff
@@ -132,8 +141,6 @@ function parse_object : (cursor : Cursor mut&) -> Optional<JSON_Node> = {
             return make<Optional<JSON_Node>>();
```

Comment:
I can't comment the previous line but you're still deallocating the vector, this should be updated to the map

Reply:
Oops, fixed